src/py21cmfast/inputs.py

```diff
@@ -821,6 +848,10 @@ class AstroParams(StructWithDefaults):
         Impact of the LW feedback on Mturn for minihaloes. Default is 22.8685 and 0.47 following Machacek+01, respectively. Latest simulations suggest 2.0 and 0.6. See Eq. XX.
     A_VCB, BETA_VCB: float, optional
         Impact of the DM-baryon relative velocities on Mturn for minihaloes. Default is 1.0 and 1.8, and agrees between different sims. See Eq. XX.
+    h_PEAK, log10_k_PEAK: double, optional
+        ETHOS parameters for dark acoustic oscillations (DAOs) in the matter power spectrum.
+        Location (10^log10_k_PEAK h/Mpc-1) and height (h_PEAK from 0 to 1) of the first DAO peak.
+        Warm dark matter (WDM) corresponds to h_PEAK=0, and log10_k_PEAK changes its mass. See Ref.~YY. (TODO!)
```

Comment:
Just noting that there's a TODO here. Is there a plan to publish first then merge?